.github/workflows/pull-request-review.yml

```diff
@@ -24,8 +24,23 @@ jobs:
               with:
                   node-version: ${{ matrix.node }}
 
+            - name: Install Python and build tools
```

Comment:
Build step is failing without these changes

Reply:
can we use https://github.com/actions/setup-python ?

Reply:
Good call. Done.